globus_sdk/transfer/client.py

```diff
@@ -334,6 +334,67 @@ def submit_delete(self, data):
         """
         return self.post('/delete', data)
 
+    def make_submit_transfer_data(self, source_endpoint, dest_endpoint,
+                                  transfer_items, label, sync_level=None,
+                                  **kwargs):
```

Comment:
Perhaps this should be `label=None`? If `label` is required without a default there might be a tendency to pass in an empty string here, and unless it's changed recently Transfer will reject an empty string `label`.